input/tar2/tar_input.go

```diff
@@ -0,0 +1,163 @@
+package tar2
+
+import (
+	"archive/tar"
+	"bytes"
+	"crypto/sha512"
+	"encoding/base64"
+	"hash"
+	"io"
+	"os"
+	"path"
```

Comment:
Does the tar spec dictate that separators are `/`? 

Point being I'd like a quick comment left somewhere; file-handling code not using `filepath` is a bit surprising, particularly when test code in the same package imports `filepath`.